patroni/config.py

```diff
@@ -198,7 +198,9 @@ def _safe_copy_dynamic_configuration(self, dynamic_configuration):
                 allowed_keys = self.__DEFAULT_CONFIG['standby_cluster'].keys()
                 expected = {
                     k: v for k, v in (value or {}).items()
-                    if (k in allowed_keys and isinstance(v, six.string_types))
+                    if (k in allowed_keys and (
+                        isinstance(v, six.string_types) or
```

Comment:
why values are being filtered at all? `self.__DEFAULT_CONFIG['standby_cluster'].keys()` is known  to have valid keys only, right?